sample-quickstart.rst

```diff
@@ -0,0 +1,217 @@
+============================
+Getting started with Flocker
+============================
+
+
+.. intro-text::
+
+   Flocker lets you run microservices apps with database containers and move them around between servers. It comes in two pieces and you’ll need both.
+
+.. contents::
+   :local:
+
+.. tutorial-step::
+
+   Step 1: Installing Flocker CLI & Node
+   =====================================
+
+   .. parallel::
+
+      .. mobile-label::
+
+         Local
+
+      |Flocker CLI diagram|
+
+      Flocker CLI
+      -----------
+
+      +--------------------------------------------------------------------------+
+      | Runs on your laptop                                                      |
+      +--------------------------------------------------------------------------+
+      | Uses application and deployment configuration files                      |
+      +--------------------------------------------------------------------------+
+      | Deploys containers to a cluster of servers                               |
+      +--------------------------------------------------------------------------+
+
+      .. container:: arrow-down center-block hidden-xs hidden-sm
+
+         .
+
+   .. parallel::
+
+      .. mobile-label::
+
+         Live
+
+      |Flocker Node diagram|
+
+      Flocker Node
+      ------------
+
+      +--------------------------------------------------------------------------+
+      | Runs on each server in a cluster                                         |
+      +--------------------------------------------------------------------------+
+      | Links, ports and volumes work across hosts                               |
+      +--------------------------------------------------------------------------+
+      | After deployment, containers can move around                             |
+      +--------------------------------------------------------------------------+
+
+      .. container:: arrow-down center-block
+
+         .
+
+   .. parallel::
+
+      .. mobile-label::
+
+         Local
+
+
+      Installation
+      ------------
+
+      .. tabs::
+
+         OS X
+         ^^^^
+
+         Install the flocker-cli client on your Mac (requires homebrew):
+
+         .. code-block:: console
+
+            $ brew update && \
+              brew tap clusterhq/flocker && \
+              brew install flocker-0.3.2
+
+         Linux
+         ^^^^^
+
+         Install the flocker-cli client on your Linux machine:
+
+         .. code-block:: console
+
+            $ virtualenv flocker-tutorial && \
+              flocker-tutorial/bin/pip install --upgrade pip && \
+              flocker-tutorial/bin/pip install --quiet flocker-cli
+
+      .. container:: arrow-down center-block invisible
```

Comment:
Yes, another problem with `container` is that it will output HTML with a `container` class on the `div`, e.g.

```
<div class="arrow-down arrow-offset center-block container"></div>
```

`container` is a reserved class name in the Bootstrap framework so this will cause display problems.